test/BUILD

```diff
@@ -184,3 +184,18 @@ python_test(
     ],
     deps = ["//third_party/python:cx_oracle"],
 )
+
+# There's no wheel naming scheme in here so this
+# will use the wheel_resolver tool
+gentest(
+    name = "simple_wheel_resolver_test",
+    srcs = [":cryptography"],
+    test_cmd = "sleep 0.1",
```

Comment:
nit: `test_cmd = "true"`

I was wondering if there was something that `sleep` was doing, but it seems that you just want to check that the test runs sucessfully